apps/zero-runtime-vite-app/vite.config.ts

```diff
@@ -18,73 +16,15 @@ theme.applyDarkStyles = function applyDarkStyles(obj) {
 
 const varPrefix = 'app';
 
-function muiZeroVitePlugin(): PluginOption {
-  function injectMUITokensPlugin(): PluginOption {
-    return {
-      name: 'vite-mui-theme-injection-plugin',
-      load(id) {
-        if (id.endsWith('@mui/zero-runtime/styles.css')) {
-          return {
-            code: generateCss(
-              {
-                cssVariablesPrefix: varPrefix,
-                themeArgs: {
-                  theme,
-                },
-              },
-              {},
-            ),
-            map: null,
-          };
-        }
-        return null;
-      },
-    };
-  }
-  const extensions = ['.ts', '.tsx', '.js', '.jsx', '.mts', '.mjs', '.cts', '.cjs', '.mtsx'];
-
-  function intermediateBabelPlugin(): PluginOption {
-    return {
-      name: 'vite-intermediate-plugin',
-      async transform(code, id) {
-        const [filename] = id.split('?');
-        if (
-          !filename.includes('zero-runtime-vite-app/src') ||
-          !extensions.some((ext) => filename.endsWith(ext))
-        ) {
-          return null;
-        }
-        const result = await transformAsync(code, {
-          filename,
-          babelrc: false,
-          configFile: false,
-          plugins: [['@mui/zero-tag-processor/pre-linaria-plugin']],
-        });
-        return {
-          code: result?.code ?? code,
-          map: result?.map,
-        };
-      },
-    };
-  }
-
-  // @TODO - Expect most of these from users of the plugin.
-  const linariaPlugin = linaria({
-    displayName: false,
-    sourceMap: true,
-    // @ts-ignore
-    cssVariablesPrefix: varPrefix,
-    themeArgs: {
-      theme,
-    },
-    babelOptions: {
-      plugins: ['@babel/plugin-syntax-jsx'],
-    },
-  });
-
-  return [injectMUITokensPlugin(), intermediateBabelPlugin(), linariaPlugin];
-}
-
 export default defineConfig({
-  plugins: [muiZeroVitePlugin(), reactPlugin(), splitVendorChunkPlugin()],
+  plugins: [
+    zeroVitePlugin({
+      cssVariablesPrefix: varPrefix,
+      themeArgs: {
+        theme,
+      },
```

Comment:
```suggestion
      theme,
```

What else would be otherwise propagate to the `themeArgs` object?

Reply:
It is just to keep the path open for future in case with want to pass any other options to the styled call.

```js
styled()(({ theme, futureArg1, futureArg2 }) => {})
```

Reply:
Even so, why would we call them `themeArgs` if they are something else? Maybe `styledArgs` would be better name, but maybe until we know for sure if there is need for it, we can keep the `theme` on the first level of the options for the plugin.